src/detectors/EEMC/CalorimeterHit_factory_EcalEndcapNRecHits.h

```diff
@@ -26,15 +26,15 @@ class CalorimeterHit_factory_EcalEndcapNRecHits : public JFactoryT<edm4eic::Calo
         m_input_tag = "EcalEndcapNRawHits";
 
         // digitization settings, must be consistent with digi class
-        m_capADC=8096;//{this, "capacityADC", 8096};
-        m_dyRangeADC=100. * MeV;//{this, "dynamicRangeADC", 100. * MeV};
-        m_pedMeanADC=400;//{this, "pedestalMean", 400};
-        m_pedSigmaADC=3.2;//{this, "pedestalSigma", 3.2};
+        m_capADC=16384;//{this, "capacityADC", 8096};
+        m_dyRangeADC=20. * GeV;//{this, "dynamicRangeADC", 100. * MeV};
```

Comment:
Where is the GeV multiplier comes from and how it is related to GeV in reco_flags.py?

Reply:
JDD4hep_service typically

Reply:
It's the same now.

Reply:
I still don't like that there is an implicit "GeV" without namespace, which means that it is getting slipped in through a header. But that is probably for another issue. 

@wdconinc

Reply:
A long list of headers have a `using namespace dd4hep`...

Reply:
The latest clang has `-Wheader-hygiene` which flags exactly this in headers (to be used with `-Werror`).